.travis.yml

```diff
@@ -87,22 +87,18 @@ install:
     # Test loadable library option
     # List of providers current as of Jan 2020
     - ./configure --prefix=$PREFIX --enable-tcp=dl
-      --disable-bgq
       --disable-efa
-      --disable-gni
       --disable-hook_debug
       --disable-mrail
       --disable-perf
       --disable-psm
```

Comment:
psm (v1) should have already be removed. so you should be able to remove `--disable-psm`